tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
+require('babel/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
Might want to mkdir -p .meteor in the script. Doesn't look like Babel will
do that for you and I don't think the dir exists in a fresh git clone.